e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -388,7 +406,9 @@ az deployment group create `
     StorageAccountName=$storageAccountName `
     KeyVaultName=$keyVaultName `
     DpsCustomAllocatorRunCsxContent=$dpsCustomAllocatorRunCsxContent `
-    DpsCustomAllocatorProjContent=$dpsCustomAllocatorProjContent
+    DpsCustomAllocatorProjContent=$dpsCustomAllocatorProjContent `
+    HubUnitsCount=$iothubUnitsToBeCreated
```

Comment:
Can you update [1N ](https://microsoft.sharepoint.com/teams/IoTDevelopersDevices/_layouts/OneNote.aspx?id=%2Fteams%2FIoTDevelopersDevices%2FShared%20Documents%2FAzure%20IoT%20Managed%20SDK%20Team%2FManaged%20SDK%20team%201N&wd=target%28Testing.one%7CA29E144A-05CF-4713-A987-F36EF7EB324D%2FIotHub%20Tests%7CAC522BC7-5108-4127-8934-3DAE6B34D3CB%2F%29
onenote:https://microsoft.sharepoint.com/teams/IoTDevelopersDevices/Shared%20Documents/Azure%20IoT%20Managed%20SDK%20Team/Managed%20SDK%20team%201N/Testing.one#IotHub%20Tests&section-id={A29E144A-05CF-4713-A987-F36EF7EB324D}&page-id={AC522BC7-5108-4127-8934-3DAE6B34D3CB}&end)to explain this new param?